plugin-locationlayer/src/main/java/com/mapbox/mapboxsdk/plugins/locationlayer/LocationLayerOptions.java

```diff
@@ -1534,6 +1564,17 @@ private Builder(LocationLayerOptions source) {
       return this;
     }
 
+    /**
+     * Sets the animation duration multiplier.
+     *
+     * @param animationDurationMultiplier the animation duration multiplier
+     * @since 0.9.0
+     */
+    public LocationLayerOptions.Builder animationDurationMultiplier(float animationDurationMultiplier) {
```

Comment:
Could we indicate that this is a tracking animation duration? Either in the method name, javadoc or both works.

Reply:
Yes, went with updating all occurrences of the name by pre-preding tracking to it so it conforms to other tracking mode configurations. Thanks for the quick 👀.